Ansible REX  test_positive_run_job fails with CLIFactoryError: Failed to create JobInvocation with data

The supplied data probably need reviewing, error output:

```
E           robottelo.cli.factory.CLIFactoryError: Failed to create JobInvocation with data:
E           {
E             "async": null,
E             "bookmark": null,
E             "bookmark-id": null,
E             "concurrency-level": null,
E             "cron-line": null,
E             "description-format": null,
E             "dynamic": null,
E             "effective-user": null,
E             "end-time": null,
E             "input-files": null,
E             "inputs": "command=\"ls\"",
E             "job-template": "Run Command - Ansible Default",
E             "job-template-id": null,
E             "max-iteration": null,
E             "search-query": "name ~ hibclwylg97ly0qh.lab.eng.rdu2.redhat.com",
E             "start-at": null,
E             "start-before": null,
E             "time-span": null
E           }
E           Command "job-invocation create" finished with return_code 70
E           stderr contains following message:
E           Task da788da6-d7d9-4538-9e15-4492838c040d planned: 0.0/1, 0%, elapsed: 00:00:00
E           Task da788da6-d7d9-4538-9e15-4492838c040d running: 0.0/1, 0%, 0.0/s, elapsed: 00:00:02
E           Task da788da6-d7d9-4538-9e15-4492838c040d running: 0.0/1, 0%, 0.0/s, elapsed: 00:00:04
E           Task da788da6-d7d9-4538-9e15-4492838c040d running: 0.0/1, 0%, 0.0/s, elapsed: 00:00:06
E           Task da788da6-d7d9-4538-9e15-4492838c040d running: 0.0/1, 0%, 0.0/s, elapsed: 00:00:08
E           Task da788da6-d7d9-4538-9e15-4492838c040d running: 0.0/1, 0%, 0.0/s, elapsed: 00:00:10
E           Task da788da6-d7d9-4538-9e15-4492838c040d warning: 1.0/1, 100%, 0.1/s, elapsed: 00:00:12
E           Task da788da6-d7d9-4538-9e15-4492838c040d warning: 1.0/1, 100%, 0.1/s, elapsed: 00:00:12
E           Error: A sub task failed
```

--- a/tests/foreman/cli/test_remoteexecution.py
+++ b/tests/foreman/cli/test_remoteexecution.py
@@ -939,15 +939,33 @@ class AnsibleREXTestCase(CLITestCase):
         super(AnsibleREXTestCase, cls).setUpClass()
         cls.sat6_hostname = settings.server.hostname
         # register and setup a host here and tests will share the host, step 0.
-        cls.org = make_org()
-        cls.client = VirtualMachine(distro=DISTRO_RHEL7)
+        cls.org = entities.Organization().create()
+        # create subnet for current org, default loc and domain
+        # add rex proxy to subnet, default is internal proxy (id 1)
+        # using API due BZ#1370460
+        cls.sn = entities.Subnet(
+            domain=[1],
+            gateway=settings.vlan_networking.gateway,
+            ipam='DHCP',
+            location=[DEFAULT_LOC_ID],
+            mask=settings.vlan_networking.netmask,
+            network=settings.vlan_networking.subnet,
+            organization=[cls.org.id],
+            remote_execution_proxy=[entities.SmartProxy(id=1)],
+        ).create()
+        # Create VM and register content host
+        cls.client = VirtualMachine(
+            distro=DISTRO_RHEL7,
+            provisioning_server=settings.compute_resources.libvirt_hostname,
+            bridge=settings.vlan_networking.bridge)
+        cls.addCleanup(vm_cleanup, cls.client)
         cls.client.create()
         cls.client.install_katello_ca()
         cls.client.register_contenthost(
             org=cls.org['label'],
             lce='Library'
         )
-        cls.assertTrue(cls.client.subscribed, True)
+        cls.assertTrue(cls.client.subscribed)
         Host.set_parameter({
             'host': cls.client.hostname,
             'name': 'remote_execution_connect_by_ip',

